### DISCUSSION TITLE
Sharpen Edges and Auto Align

### DISCUSSION THREAD

**Discussion by kaelsonofkrypto** [nagadomi/nunif#94] [General]
Hi can a sharpen edges and auto align based off the edges  option be added. This would allow better accuracy anytime things would be off a fraction of a pixel, this is especially useful to check when doing an auto converge. 3D combine has an option like this that reduces artifacts, but i think iw3 would be able to do this better since the 3d of iw3 looks better. These two functions being added would help iw3 reduce artifacts i believe. Thanks

**Comment by loawizard** [comment]
Would be awesome
